integration-tests/bats/sql-server.bats

```diff
@@ -850,6 +850,51 @@ SQL
     )' ""
 
     server_query repo1 1 "SHOW tables" "" # no tables on master
-    
+
     server_query "repo1/feature-branch" 1 "SHOW Tables" "Table\ntest"
 }
+
+@test "sql-server: SET GLOBAL default branch" {
+    skiponwindows "Has dependencies that are missing on the Jenkins Windows installation."
+
+    cd repo1
+    dolt checkout -b "new"
+    dolt checkout master
+    start_sql_server repo1
+
+    multi_query repo1 1 '
+    select dolt_checkout("new");
+    CREATE TABLE t (a int primary key, b int);
+    INSERT INTO t VALUES (2,2),(3,3);' ""
+
+    server_query repo1 1 "SHOW tables" "" # no tables on master
+
+    server_query repo1 1 "set GLOBAL dolt_default_branch = 'refs/heads/new';" ""
+    server_query repo1 1 "select @@GLOBAL.dolt_default_branch;" "@@GLOBAL.dolt_default_branch\nrefs/heads/new"
+    server_query repo1 1 "select active_branch()" "active_branch()\nnew"
+    server_query repo1 1 "SHOW tables" "Table\nt"
+}
+
+@test "sql-server: SET GLOBAL default branch does not affect current session" {
+    skiponwindows "Has dependencies that are missing on the Jenkins Windows installation."
+
+    cd repo1
+    dolt checkout -b "new"
+    dolt checkout master
+    start_sql_server repo1
+
+    multi_query repo1 1 '
+    select dolt_checkout("new");
+    CREATE TABLE t (a int primary key, b int);
+    INSERT INTO t VALUES (2,2),(3,3);' ""
+
+    server_query repo1 1 "SHOW tables" "" # no tables on master
+
+    multi_query repo1 1 '
+    set GLOBAL dolt_default_branch = "refs/heads/new";
+    show tables;' "" # SET GLOBAL does not affect current connection
```

Comment:
I could be wrong, but I think that multi_query doesn't actually do any checks with the result string given. Try it and see

Reply:
yeah multi_query doesn't pass that arg

Reply:
i'll just delete, idk how to test within current session